Classes/Package.php

```diff
@@ -1,37 +0,0 @@
-<?php
-namespace Neos\EventSourcing;
-
-/*
- * This file is part of the Neos.EventSourcing package.
- *
- * (c) Contributors of the Neos Project - www.neos.io
- *
- * This package is Open Source Software. For the full copyright and license
- * information, please view the LICENSE file which was distributed with this
- * source code.
- */
-
-use Neos\EventSourcing\Projection\Doctrine\DoctrineProjectionPersistenceManager;
-use Neos\Flow\Core\Bootstrap;
-use Neos\Flow\Package\Package as BasePackage;
-
-/**
- * The Neos.EventSourcing Package
- */
-class Package extends BasePackage
-{
-    /**
-     * @var boolean
-     */
-    protected $protected = true;
-
-    /**
-     * @param Bootstrap $bootstrap The current bootstrap
-     * @return void
-     */
-    public function boot(Bootstrap $bootstrap)
-    {
-        $dispatcher = $bootstrap->getSignalSlotDispatcher();
-        $dispatcher->connect(Bootstrap::class, 'finishedRuntimeRun', DoctrineProjectionPersistenceManager::class, 'persistAll');
```

Comment:
`DoctrineProjectionPersistenceManager` was renamed to `DoctrineProjectionState` and is no longer a "singleton".
The `persistAll()` call is now invoked in the `shutdownObject()` lifecycle method of that class